### PR TITLE
qick fix

### PR DESCRIPTION
**Pull request by sulochanaviji**
good

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -46,7 +46,7 @@ playground.xcworkspace
 # you should judge for yourself, the pros and cons are mentioned at:
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 #
-Pods/
+# Pods/
```
Comment (raysarebest) [inline .gitignore:49]:
```suggestion
Pods/
```

Some of our pods haven't been updated for Swift 4.2 yet, and we've made those fixes ourselves directly in this repository while we investigate other options, so we're committing our `Pods` directory for now

File: WHIR/ViewControllers/TableViewController.swift [hunk 2]
```diff
@@ -12,40 +12,66 @@ import BarcodeScanner
...
+    }
+
+    func setupIntents() {
+        let activity = NSUserActivity(activityType: "dirkhulverscheidt.WHIR.addBook")
+        activity.title = NSLocalizedString("addBook", comment: "Add Book")
+        activity.userInfo = [ : ]
+        activity.isEligibleForSearch = true
+        if #available(iOS 12.0, *) {
+            activity.isEligibleForPrediction = true
+        } else {
```
Comment (raysarebest) [inline WHIR/ViewControllers/TableViewController.swift:36]: The `else` block here is empty. Do we really need it?

File: WHIR/Views/sv.lproj/About.strings [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
Barcode icon made by Pixel perfect on Flaticon (https://www.flaticon.com/free-icon/barcode_726558).
\nUsing CocoaPod KMPlaceholderTextView by MoZhouqi (https://github.com/MoZhouqi/KMPlaceholderTextView). Licensed under the MIT license.\n\nThanks for reading and using this App ❤.\n\nFeel free to contact me on my support email (can be found on the App Store page), if you find bugs or just to say hello. (br.apps@icloud.com)
"; ObjectID = "lpw-Ku-zCI"; */
+"lpw-Ku-zCI.text" = "Handgjord av © 2018 Bruce Röttgers och flera andra på GitHub\n\nLogo av Zlatko Najdenovski på Flaticon. Licensierad enligt Flaticon Basic License (https://www.flaticon.com/authors/zlatko-najdenovski).\n
Streckkod ikon gjord av Pixel perfect på Flaticon (https://www.flaticon.com/free-icon/barcode_726558).
\nAnvänder CocoaPod KMPlaceholderTextView av MoZhouqi (https://github.com/MoZhouqi/KMPlaceholderTextView). Licensierad enligt MIT license.\n\nTack för att du läste och använde den här appen ❤.\n\nKänna sig fri att kontakta mig med min support e-postadress (kan hittas på den App Store sida), om du hittar buggar eller bara att säga hej. (br.apps@icloud.com)
";
```
Comment (raysarebest) [inline WHIR/Views/sv.lproj/About.strings:6]: I translated this string into Swedish, and it's currently in `master`. What happened to it?

**Review by raysarebest — CHANGES_REQUESTED** [review]

**Review by bcye — CHANGES_REQUESTED** [review]
I don't understand what you did. Can you please explain what you're trying to resolve with your pull request

**Pull request closed without merging**